scripts/activiti/01_active_workflows.sql

```diff
@@ -0,0 +1,11 @@
+-- A query to determine whether there are active activiti workflows lurking somewhere.
+-- Under normal circumstances, a workflow will be terminated after completion of a process or it will timeout after 24 hours have elapsed since it was started.
+-- If a workflow has been active for more than 24 hours that's a definite sign of a deeper problem!! See ElapsedTime field in the results.
+
+SELECT act.proc_inst_id_ as "ProcessInstanceId", def.name_ as "Name", act.start_time_ as "ActivityStartTime",  age(now()::timestamp, act.start_time_) as "ElapsedTime",
```

Comment:
Looks good. Two questions:

-  Do you have an example of a failure so we can verify that it works as expected
-  Should we not add an additional condition to the where so it will only return those that exceed 24h?

Reply:
- A quick test example is to edit a project/budget on the front end and click cancel. The workflow that was started will remain active even after exiting the WIP module.
- We will modify the script once we've seen the actual behaviour when testing TA8-655. For now I want to see that there are no active workflows after a budget pull/import is finished for example.

Reply:
Happy